frontend/src/admin/components/sidebar/dropdown/dropdown-menu-item.scss

```diff
@@ -0,0 +1,19 @@
+@import "src/styles/main";
+
+p {
+  color: $contrast-color;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  padding: 1rem;
```

Comment:
You are mixing `rem` and `px`. We should stick to just use one of them

Reply:
✅ is fixed